cli/tests/pcluster/templates/test_login_nodes_stack/test_login_nodes_dna_json/dna-1.json

```diff
@@ -9,6 +9,7 @@
     "custom_awsbatchcli_package": "",
     "custom_node_package": "",
     "cw_logging_enabled": "true",
+    "default_user_home": "shared",
```

Comment:
Nitpick: Can we test with the other default value in one of the unit tests too. Like `dna-1.json` with `local` and`dna-2.json` with shared

Reply:
good idea, thanks!